src/seedu/addressbook/commands/Command.java

```diff
@@ -36,6 +36,10 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
         return String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, personsDisplayed.size());
     }
 
+    public static String getMessageForSortedPersonListShownSummary(List<? extends ReadOnlyPerson> personsDisplayed) {
```

Comment:
Missing header comments, do remember to follow the coding standards to improve the readability of your codes.